packages/fast-usdc/src/type-guards.js

```diff
@@ -26,3 +28,10 @@ export const makeProposalShapes = ({ PoolShares, USDC }) => {
   });
   return harden({ deposit, withdraw });
 };
+
+const NatAmountShape = { brand: BrandShape, value: M.nat() };
+/** @type {TypedPattern<FastUsdcTerms>} */
+export const FastUSDCTermsShape = harden({
```

Comment:
this name isn't technically correct, since brands are also a term. That's why `meta` has the awkward `customTermsShape`. 

I'm guessing you pulled this out so you could annotate a `TypedPattern`. I think you could do that inline. Then `meta.customTermsShape` could be the canonical place to get a contract's terms. I advocate for that over naming another constant like this.

Reply:
But then, to import this value, you drag in the whole contract.

Reply:
as to the name, it follows the precedent of `FastUsdcTerms`. In both cases, it's conventional to read them as "the terms that are custom to Fast USDC".